client/state/comments/actions.js

```diff
@@ -221,6 +222,13 @@ export function writeComment( commentText, siteId, postId, parentCommentId ) {
 				skipSort
 			} );
 
+			dispatch( {
+				type: COMMENTS_COUNT_INCREMENT,
+				siteId,
+				postId,
+				incrementBy: 1
```

Comment:
In this case, yes. This dispatch is happening inside the `writeComment` function, which adds a new comment when it's added by a user in `reader/comments/form.jsx`.